presto-product-tests/bin/run_on_docker.sh

```diff
@@ -114,42 +114,46 @@ function terminate() {
   exit 130
 }
 
+function getAvailableEnvironments() {
+AVAILABLE_ENVIRONMENTS=`ls $ENVIRONMENT_LOCATION -I files`
+}
+
 ENVIRONMENT=$1
+SCRIPT_DIR=$(dirname $(absolutepath "$0"))
+PRODUCT_TESTS_ROOT="${SCRIPT_DIR}/.."
+PROJECT_ROOT="${PRODUCT_TESTS_ROOT}/.."
+ENVIRONMENT_LOCATION="${PRODUCT_TESTS_ROOT}/conf/docker/"
+
+# Get the list of valid environments
+getAvailableEnvironments
 
-if [[ "$ENVIRONMENT" != "singlenode" && "$ENVIRONMENT" != "multinode" && "$ENVIRONMENT" != "singlenode-kerberized" ]]; then
-   echo "Usage: run_on_docker.sh <singlenode|multinode|singlenode-kerberized> <product test args>"
+if [[ ! -d "$ENVIRONMENT_LOCATION$ENVIRONMENT" ]]; then
```

Comment:
I think you can init the `ENVIRONMENT_LOCATION` without trailing slash, and change this like to be `$ENVIRONMENT_LOCATION/$ENVIRONMENT`. I believe it is more readable, because 2 variables are not glued together.